src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -62,6 +62,9 @@ bool FrontEnd::supported_impl(const std::vector<ov::Any>& variants) const {
         std::string model_path = variants[0].as<std::string>();
         if (ov::util::ends_with(model_path, suffix.c_str())) {
             return true;
+        } else if (ov::util::directory_exists(model_path) && ov::util::file_exists(model_path + "/saved_model.pb") &&
+                   ov::util::file_exists(model_path + "/variables/variables.index")) {
```

Comment:
Also, can you please implement separate method to check if it is SavedModel format? After that, you can use it in both `load_impl` and `supported_iml` because the checks are the same but in one of them it is incomplete.

Reply:
Done.